### DISCUSSION TITLE
Sort by date added

### DISCUSSION THREAD

**Discussion by hinogi** [dailydotdev/daily#395] [Feature Requests]
Somehow I currently seem to be locked in using predefined sorting options when I just want to see the latest articles but I cannot.

**Comment by idoshamun** [comment]
Hi @hinogi,

Are you looking for the recent feed?
We removed it in the last version and are now reconsidering reverting it due to the feedback we received.

**Reply by hinogi** [reply]
Exactly, because all the other options may have impact on the order the articles are shown, and that messes with my spatial perception memory

**Reply by hinogi** [reply]
Maybe even add custom sort orders or grouping options, by day/month or issuer or so

**Reply by mrevjd** [reply]
I found the recent feed the most useful.
https://github.com/dailydotdev/daily/discussions/392#discussion-3763727

**Reply by idoshamun** [reply]
@hinogi following your way of thinking, if we keep the order for your popular feed this should do it, right?
For example, you will see at the top all the articles that you weren't exposed to in your last session, but our popular ranking, and it will be followed by articles that you saw in the order you are familiar with. Does that make sense to you?
And in any way we will consider adding back the recent feed.

**Reply by svg153** [reply]
I found more information in #402
As @idoshamun said... It is [already in development](https://github.com/dailydotdev/daily/discussions/402#discussioncomment-1963303).

**Reply by nimrodkra** [reply]
@hinogi @mrevjd @svg153 the change is live: https://app.daily.dev/posts/e-dQ6UyIA
